packages/react-components/src/Autocomplete/autocomplete.tsx

```diff
@@ -0,0 +1,689 @@
+}>((props) => ({
+  label: 'Autocomplete-tag',
+  borderRadius: '3px',
+  margin: 0,
+  ...(props.tagsLength === 1 && {
+    maxWidth: 'calc(100% - var(--pv-size-base))',
+  }),
+  ...(props.tagsLength > 1 && props.limitTags > 0 && {
+    maxWidth: `calc(${100 / props.limitTags}% - var(--pv-size-base))`,
+  }),
+  ...(props.size === 'small' && {
+    height: 'var(--pv-size-base-5)',
+  }),
+}));
+
+const AutocompleteTagSize = styled(Typography)({
+  margin: '0 var(--pv-size-base-2)',
+});
+
+const AutocompleteInputField = styled(Typography)(
+  () => ({
+    fontFamily: 'inherit',
+    outline: 'none',
+    boxSizing: 'border-box',
+    minWidth: '30px',
+    width: 0,
+    flexGrow: 1,
+    backgroundColor: 'transparent',
+    borderStyle: 'none',
+    appearance: 'none',
+  }),
+  (props) => {
+    const isDark = props.theme.mode === 'dark';
+    const color = isDark
+      ? 'var(--pv-color-white)'
+      : 'var(--pv-color-black)';
+
+    let colorPlaceholder = 'var(--pv-color-gray-9)';
+    let colorDisabled = 'var(--pv-color-gray-7)';
+
+    if (isDark) {
+      colorPlaceholder = 'var(--pv-color-gray-6)';
+      colorDisabled = 'var(--pv-color-gray-4)';
+    }
+
+    return ({
+      color,
+      '&::placeholder': {
+        color: colorPlaceholder,
+      },
+      '&:disabled': {
+        cursor: 'not-allowed',
+        color: colorDisabled,
+      },
+    });
+  },
+);
+
+const AutocompleteError = styled(Typography)({
+  marginTop: '2px',
+});
+
+const AutocompleteLabel = styled('label')({
+  label: 'TextField-label',
+  marginBottom: '2px',
+  display: 'inline-block',
+});
+/**
+ *
+ */
+
+export const Autocomplete = <
+  T,
+  Multiple extends boolean | undefined = false,
+>(props: AutocompleteOwnProps<T, Multiple>): JSX.Element => {
+  const {
+    className,
+    size,
+    placeholder,
+    label,
+    disabled = false,
+    noOptionsText,
+    loading,
+    loadingText,
+    limitTags = -1,
+    name,
+    required,
+    multiple = false,
+    readOnly,
+    error,
+    errorText,
+    renderRoot: renderRootProp,
+    renderOption: renderOptionProp,
+    getLimitTagsText = (more) => `${more} more`,
+    groupBy,
+    onCreate,
+  } = props;
+  const {
+    id,
+    value,
+    searchValue,
+    groupedOptions,
+    getRootProps,
+    getInputLabelProps,
+    getInputProps,
+    getListboxProps,
+    getOptionProps,
+    getPopoverProps,
+    getTagProps,
+    getOptionLabel,
+    handleDeleteAllValues,
+  } = useAutocomplete(props);
+  const {
+    onChange,
+    ...otherInputProps
+  } = getInputProps();
+
+  const rootProps = getRootProps();
+  const popoverProps = getPopoverProps();
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    onChange(event);
+
+    const { value: valueInput } = event.target;
+
+    if (valueInput === '') {
+      handleDeleteAllValues(event);
+    }
+  };
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
When the tab is used, the component menu opens and stops working with the keyboard.
Reproduce:
1. Make sure that the component is not in focus, click on tab. The component must be in focus.
2. Click the tab again. The component menu opens, the focus disappears and the component itself does not respond to other keys.